index.html

```diff
@@ -2,21 +2,19 @@
 <html lang="en">
     <head>
         <meta charset="utf-8" />
-        <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+        <link rel="icon" href="/src/favicon.ico" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <meta name="theme-color" content="lightgray" />
-        <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
-        <link rel="stylesheet" href="https://unpkg.com/higlass@1.11.3/dist/hglib.css">
+        <link rel="stylesheet" href="https://unpkg.com/higlass@1.12/dist/hglib.css">
         <script src="https://cdn.tailwindcss.com"></script>
         <style>
             /* To resolve style conflicts with Tailwind */
             .gosling-component { height: auto !important; }
         </style>
-        <title>Gosling.js React Example</title>
+        <title>Gosling React Vite Example</title>
```

Comment:
I think it's fine to leave as "Gosling React Example". Vite is just a build tool.